applications/api-cat/src/main/java/no/acat/repository/ApiDocumentRepository.java

```diff
@@ -0,0 +1,35 @@
+package no.acat.repository;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import no.acat.model.ApiDocument;
+import no.acat.service.ElasticsearchService;
+import org.elasticsearch.action.get.GetResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.Optional;
+
+/*
+ApiDocumentRepository is a Repository abstraction over elasticsearch storage implementation
```

Comment:
Glimrende med kommentar! Gjør det mye lettere for andre å få med seg sammenhengen.